rest-api/api/awardee_api.py

```diff
@@ -0,0 +1,19 @@
+from api.base_api import BaseApi
+from app_util import auth_required
+from api_util import PTC_AND_HEALTHPRO
+from dao.hpo_dao import HPODao
+from werkzeug.exceptions import NotFound
+
+class AwardeeApi(BaseApi):
```

Comment:
Test for this API?

Reply:
Working on it. :) (The start of it is in this commit.)